mscore/fretproperties.cpp

```diff
@@ -343,7 +343,7 @@ void FretCanvas::mousePressEvent(QMouseEvent* ev)
                               };
 
                               // Find the lowest dot type that doesn't already exist on the string
-                              for (int i = 0; i < dtypes.size(); i++) {
+                              for (int i = 0; i < int(dtypes.size()); i++) {
```

Comment:
I know std::vector::size doesn't necessarily return int, but it won't _not_ work without this conversion, though? Basically, is it really necessary?

Reply:
it returns a size_t, which is an unsigned type. Alternative woule be to change to `for (size_t i = 0; i < dtypes.size(); i++) {`

Reply:
"_won't not work without_" too much double negate for me to parse ;-)

Reply:
Actually:
Return Value
The number of elements in the container.

Member type `size_type` is an unsigned integral type.

(from http://www.cplusplus.com/reference/vector/vector/size/)

Reply:
Knowing that the vector in question here has just 4 elements makes that cast to int pretty safe though ;-)

Reply:
I would say using 'size_t' with stl containers as a type of the index should become a part of the code style.

Reply:
OK, next PR ;-)